akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2Multiplexer.scala

```diff
@@ -42,7 +42,7 @@ private[http2] trait Http2Multiplexer {
  */
 @InternalApi
 private[http2] trait Http2MultiplexerSupport { logic: GraphStageLogic with StageLogging =>
-  def createMultiplexer(outlet: GenericOutlet[FrameEvent], prioritizer: StreamPrioritizer): Http2Multiplexer =
+  def createMultiplexer(outlet: GenericOutlet[FrameEvent], prioritizer: StreamPrioritizer, http2StreamHandling: Http2StreamHandling): Http2Multiplexer =
```

Comment:
(Information of closing 'outgoing' streams must somehow be passed from the multiplexer (which deals with 'outgoing' streams) to the stream handling (which deals with 'incoming' streams), but passing in `Http2StreamHandling` in wholesale is not very elegant)

Reply:
It's a mixture of cake pattern and composition, so we could also add abstract internal SPI methods to `Http2MultiplexerSupport` and implement them in `Http2StreamHandling`.